Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -0,0 +1,76 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import WebKit
+
+// TODO: RUMM-1794 rename the method
+public extension WKUserContentController {
+    func addDatadogMessageHandler(allowedWebViewHosts: [String]) {
+        let bridgeName = DatadogMessageHandler.name
```

Comment:
Why using a static `DatadogMessageHandler.name`? I don't see it being used outside of this method.

Reply:
i thought `WKUserContentController` should know as little as possible about `DatadogMessageHandler` internals, but this is a trivial change and doesn't change much as you noticed.

should i change anyway?

Reply:
Yea not a big deal, makes it less easy to read maybe. Up to you!